### ISSUE TITLE
Object->get_information($id) doesn't work with non-existant $id

### ISSUE THREAD

**Issue by matwei** [matwei-perl/WebService-IdoitAPI#6]
The error message is:

```
Not a HASH reference at /home/weidnerm/perl5/lib/perl5/WebService/IdoitAPI/Object.pm line 91.
 at /home/weidnerm/perl5/lib/perl5/WebService/IdoitAPI/Object.pm line 91.
        WebService::IdoitAPI::Object::get_information(WebService::IdoitAPI::Object=HASH(0x7fffb9905300), undef) called at /home/weidnerm/perl5/lib/perl5/WebService/IdoitAPI/Object.pm line 28

```
This is because in line 91 is a reference to `$obj->{objecttype}`, but `$obj` is `undef` when the ID is invalid.

**Comment by matwei** [comment]
This also concerns Object->get_data() because it uses get_information()

**Comment by matwei** [comment]
Fixed in d6edb9b
